src/NServiceBus.AcceptanceTests/NServiceBus.AcceptanceTests.csproj

```diff
@@ -40,6 +40,7 @@
     <_PackageFiles Remove="**\obj\**\*.cs" />
     <_PackageFiles Remove="Core\**\*.cs" />
     <_PackageFiles Remove="AssemblyInfo.cs" />
+    <_PackageFiles Remove="EndpointTemplates\DefaultServerWithDiagnostics.cs" />
```

Comment:
Why this been removed from the package? Would a downstream want to be able to test an addition to the diagnostics?